Jellyfin.ApiClient/ApiClient.cs

```diff
@@ -2487,20 +2406,21 @@ public async Task<PlaylistCreationResult> CreatePlaylist(PlaylistCreationRequest
                 throw new ArgumentNullException("must provide either MediaType or Ids");
             }
 
-            var queryString = new QueryStringDictionary();
-
-            queryString.Add("UserId", request.UserId);
-            queryString.Add("Name", request.Name);
+            var queryString = new NameValueCollection
+            {
+                { "UserId", request.UserId.ToString() },
+                { "Name", request.Name }
+            };
 
             if (!string.IsNullOrEmpty(request.MediaType))
                 queryString.Add("MediaType", request.MediaType);
 
             if (request.ItemIdList != null && request.ItemIdList.Any())
-                queryString.Add("Ids", request.ItemIdList);
+                queryString.Add("Ids", request.ItemIdList.Select(o => 0.ToString()).ToList());
```

Comment:
The zero looks like a typo.

Reply:
Was a typo, fixed when I touched that in 93bfb86